src/MediaPlayer/MPNowPlayingInfoCenter.cs

```diff
@@ -32,6 +32,16 @@ public MPNowPlayingInfo ()
 		public double? PlaybackDuration;
 		public double? DefaultPlaybackRate;
 
+		// iOS 10
```

Comment:
we should have used properties (but we did not). Still this needs the availability attributes otherwise developers won't know if the field can be set on older versions of iOS